stable/redis-ha/templates/redis-ha-statefulset.yaml

```diff
@@ -147,6 +150,25 @@ spec:
           name: data
         - mountPath: /probes
           name: probes
+{{- if .Values.exporter.enabled }}
+      - name: redis-exporter
+        image: "{{ .Values.exporter.image }}:{{ .Values.exporter.tag }}"
+        imagePullPolicy: {{ .Values.exporter.pullPolicy }}
+        args:
+        {{- range $key, $value := .Values.extraArgs }}
+          - --{{ $key }}={{ $value }}
+        {{- end }}
+        env:
+          - name: REDIS_ADDR
+            value: redis://localhost:6379
+        {{- if .Values.auth }}
+          - name: REDIS_PASSWORD
+            value: {{ .Values.redisPassword }}
```

Comment:
The password should pull from the `Secret` so the password isn't stored in plain text in the definition and that way `.Values.existingSecrets` can be used.
```
          valueFrom:
            secretKeyRef:
            {{- if .Values.existingSecret }}
              name: {{ .Values.existingSecret }}
            {{- else }}
              name: {{ template "redis-ha.fullname" . }}
            {{- end }}
              key: auth
```

Reply:
Done